site/app/libraries/database/DatabaseQueries.php

```diff
@@ -836,6 +836,11 @@ public function getAuthorUserGroups($author_ids) {
         return $this->course_db->rows();
     }
 
+    public function getAuthorUserGroup($author_id) {
+        $this->course_db->query("SELECT user_id, user_group FROM users WHERE user_id = ?", [$author_id]);
+        return $this->course_db->row();
+    }
+
```

Comment:
This function is essentially a duplicate of the function above it and should be deleted.

Reply:
The function above takes as an argument multiple author IDs and returns an array of rows, and I only have one post with one author ID.